### DISCUSSION TITLE
requireJS - UMD Wrapper. __extends is not a function when loading bundled typescript through requirejs

### DISCUSSION THREAD

**Discussion by AccessAdam** [twilio-labs/paste#2850] [General] [answered]
Has anyone else hit these issues when using paste with requireJS?

-	https://github.com/evanw/esbuild/issues/2259
-	https://github.com/evanw/esbuild/issues/1348

It looks like all of the the compiled paste libraries contain this UMD wrapper and there are no bundles that use tslib.es6 from what I can see.

Are there any builds of paste that use tslib.es6 instead of the UMD wrapper?

**Comment by TheSisb** [comment]
Hi @AccessAdam,

I don't believe we've gotten any internal (Twilio) requests for RequireJS support, so unfortunately this falls pretty low on our priority list. I'd be open to reviewing a PR if you wanted to contribute a fix. Sorry for the less than ideal answer.

**Comment by SiTaggart** [comment, marked as answer]
Hi @AccessAdam 

I know this is an old issue, but we've released a new version of Paste core which should finally address this issue. Sorry it took so long.
